index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <script src="vendor.js" defer></script>
```

Comment:
Why `defer` is here ??? @Rehan-Khan-Engineer

Reply:
defer here will tell the browser to download the scripts right away when they are met, but DO NOT BLOCK PARSING HTML & execute scripts only after everything in HTML has been parsed! @Abbas4tech